### PR TITLE
feat(aleph-zero): add token definitions

### PR DESCRIPTION
**Pull request by james-a-morris**
Adds token definitions for AlephZero. The ETH/WETH addresses are handled like MATIC for non-native ETH gas token

### CODE REVIEW

File: src/tokens.ts [hunk 1]
```diff
@@ -93,6 +93,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "ETH",
     decimals: 18,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xD648529D4803d3467bA8850577BEd4e4b8Ae583C",
```
Comment (pxrl) [inline src/tokens.ts:96]: This address seems to be USDT: https://evm-explorer.alephzero.org/address/0xD648529D4803d3467bA8850577BEd4e4b8Ae583C

File: src/tokens.ts [hunk 2]
```diff
@@ -275,6 +276,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "USDT",
     decimals: 6,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0x4Ca4B85Ead5EA49892d3a81DbfAE2f5c2F75d53D",
```
Comment (pxrl) [inline src/tokens.ts:279]: This one is USDC: https://evm-explorer.alephzero.org/address/0x4Ca4B85Ead5EA49892d3a81DbfAE2f5c2F75d53D

File: src/tokens.ts [hunk 3]
```diff
@@ -323,6 +325,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "WETH",
     decimals: 18,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xD648529D4803d3467bA8850577BEd4e4b8Ae583C",
```
Comment (pxrl) [inline src/tokens.ts:328]: Same as above - this is USDT.

**Review by pxrl — COMMENTED** [review]
@james-a-morris Are you sure on these addresses? They all map to different tokens in the A0 explorer.

File: src/tokens.ts (hunk 1 above)
Comment (melisaguevara) [inline src/tokens.ts:96]: Not sure why but seems like we mess up the addresses.
This should be ETH/WETH: https://evm-explorer.alephzero.org/token/0xB3f0eE446723f4258862D949B4c9688e7e7d35d3

```suggestion
      [CHAIN_IDs.ALEPH_ZERO]: "0xB3f0eE446723f4258862D949B4c9688e7e7d35d3",
```

Comment (james-a-morris) [reply src/tokens.ts]: Done - this was a copy paste issue during a pair programming session - the corrected version was on local 😓. Thanks for the callout!

File: src/tokens.ts (hunk 2 above)
Comment (james-a-morris) [reply src/tokens.ts]: Done - this was a copy paste issue during a pair programming session - the corrected version was on local 😓. Thanks for the callout!

**Review by melisaguevara — APPROVED** [review]

**Comment by james-a-morris** [comment]
Done - this was a copy paste issue during a pair programming session - the corrected version was on local 😓. Thanks for the callout!

**Review by pxrl — APPROVED** [review]
LGTM

**Pull request merged**